### PR TITLE
Testing guide (#194)

### LINKED ISSUE #194: Add documentation to the tests directory describing how to run the tests
### Description

There is currently no documentation in the tests directory describing how to run the tests (e.g. need to run git lfs pull to download the VTK files).

### Reproduction

Try to read the documentation in the tests directory.

### Expected behavior

Have documentation in the tests directory.

### Additional context

A user tried to run a test in `tests/cases/heats/diffusion_line_source` but it failed with a VTK parse error
```
~/Downloads/svFSIplus-main/tests/cases/heats/diffusion_line_source$ ./svFSI svFSI_GMRES.xml
2024-02-21 17:55:43.085 (   0.115s) [    7F42D7843C80]       vtkXMLParser.cxx:364    ERR| vtkXMLDataParser (0x555d302759d0): Error parsing XML in stream at line 1, column 0, byte index 0: syntax error
2024-02-21 17:55:43.085 (   0.115s) [    7F42D7843C80]       vtkXMLReader.cxx:510    ERR| vtkXMLUnstructuredGridReader (0x555d3021e4d0): Error parsing input file.  ReadXMLInformation aborting.
2024-02-21 17:55:43.085 (   0.116s) [    7F42D7843C80]       vtkExecutive.cxx:729    ERR| vtkCompositeDataPipeline (0x555d3021c460): Algorithm vtkXMLUnstructuredGridReader (0x555d3021e4d0) returned failure for request: vtkInformation (0x555d3022e210)
  Debug: Off
  Modified Time: 93
  Reference Count: 1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

### PR DESCRIPTION
**Pull request by mrp089**
## Current situation
Closes #194.

## Release Notes 
I added a guide on how to run and create tests.

## Documentation
This is it.

## Testing
I locally built doxygen. Looks good and links work.

## Code of Conduct & Contributing Guidelines 
- [x] I agree to follow the [Code of Conduct](https://github.com/SimVascular/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/SimVascular/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mrp089** [comment]
@ktbolt, ready for you!

File: tests/testing.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@page testing Testing Guide
+
+[TOC]
+
+[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. Essentially, assume that every untested line in `svFSIplus` is not working correctly. Below is a quick guide on how to run and add integration tests for `svFSI`.
```
Comment (ktbolt) [inline tests/testing.md:5]: I suggest replacing
```
Essentially, assume that every untested line in `svFSIplus` is not working correctly.`
```

with something like
```
Integration testing is performed when integrating code changes into the main development branch to verify that the code works as expected.

**Comment by ktbolt** [comment]
@mrp089 Essentially, very nice!

File: tests/testing.md (hunk 1 above)
Comment (mrp089) [reply tests/testing.md:5]: Haha, I'll change it! My other idea was "always be vigilant"

**Comment by mrp089** [comment]
Should be good!

**Review by ktbolt — APPROVED** [review]
Good!

**Pull request merged**